### DISCUSSION TITLE
My TSML has crashed? "Meetings" won't load.

### DISCUSSION THREAD

**Discussion by AASTL** [code4recovery/12-step-meeting-list#1547] [General]
Hello,

My meeting list is no longer loading, at https://aastl.org/meetings/

I get an error message, "File Not Found."

At first, I thought maybe the server is down, that holds all of the meeting data, but other TSML meeting guides are still working (Dallas, Minneapolis). The rest of our website appears to be ok, just the TSML is down.

At my level of knowledge, I have no idea why this is happening, nor whom to ask for help? Could any of the volunteers take a look, and see if you can understand the issue?

Thank you!
Tom

**Comment by joshreisner** [comment]
hi tom, i think this was happening on your site persistently [about 18 months ago](https://github.com/code4recovery/12-step-meeting-list/discussions/1126) - here's a quick fix: https://github.com/code4recovery/12-step-meeting-list/discussions/1126#discussioncomment-5851958

(wordpress settings > permalinks > click save)

it would be great to find out why this is happening. did the site get restored from backup? that's the only scenario i'm aware of that can trigger this

**Comment by AASTL** [comment]
Hi Josh!



When I went to my Wordpress dashboard to see if I could find anything that
might explain the problem, I noticed that all of my version (Wordpress and
various plugins) are up to date. That was not the case a week ago, so I
wondered if there was an auto-update, or something, that caused a conflict?
I have not made any changes, not even plugin updates, except to update some
meeting info.



Is it possible that the website hosting server crashed, and the change to a
backup server is the restoration you’re talking about?



Anthony says he can take a look in just a little while!



Thank you!

Tom



*From:* Josh Reisner ***@***.***
*Sent:* Wednesday, October 23, 2024 5:02 PM
*To:* code4recovery/12-step-meeting-list
*Cc:* AASTL; Author
*Subject:* Re: [code4recovery/12-step-meeting-list] My TSML has crashed?
"Meetings" won't load. (Discussion #1547)



hi tom, i think this was happening on your site persistently about 18
months ago
<https://github.com/code4recovery/12-step-meeting-list/discussions/1126> -
here's a quick fix: #1126
<https://github.com/code4recovery/12-step-meeting-list/discussions/1126>

it would be great to find out why this is happening. did the site get
restored from backup? that's the only scenario i'm aware of that can
trigger this

—
Reply to this email directly, view it on GitHub
<https://github.com/code4recovery/12-step-meeting-list/discussions/1547#discussioncomment-11034268>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/BJLYSZLOMCDVQVHNYSDD3D3Z5AMEDAVCNFSM6AAAAABQPXEIJ2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBTGQZDMOA>
.
You are receiving this because you authored the thread.[image: Image
removed by sender.]Message ID:
<code4recovery/12-step-meeting-list/repo-discussions/1547/comments/
***@***.***>

**Comment by AASTL** [comment]
Josh,



That solution worked again!



Anthony has access to my CPanel from the webhost – maybe he can see what
happened, there?



Many, many thanks!

-Tom



*From:* Josh Reisner ***@***.***
*Sent:* Wednesday, October 23, 2024 5:02 PM
*To:* code4recovery/12-step-meeting-list
*Cc:* AASTL; Author
*Subject:* Re: [code4recovery/12-step-meeting-list] My TSML has crashed?
"Meetings" won't load. (Discussion #1547)



hi tom, i think this was happening on your site persistently about 18
months ago
<https://github.com/code4recovery/12-step-meeting-list/discussions/1126> -
here's a quick fix: #1126
<https://github.com/code4recovery/12-step-meeting-list/discussions/1126>

it would be great to find out why this is happening. did the site get
restored from backup? that's the only scenario i'm aware of that can
trigger this

—
Reply to this email directly, view it on GitHub
<https://github.com/code4recovery/12-step-meeting-list/discussions/1547#discussioncomment-11034268>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/BJLYSZLOMCDVQVHNYSDD3D3Z5AMEDAVCNFSM6AAAAABQPXEIJ2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBTGQZDMOA>
.
You are receiving this because you authored the thread.[image: Image
removed by sender.]Message ID:
<code4recovery/12-step-meeting-list/repo-discussions/1547/comments/
***@***.***>
